bootstrap-pricing-page.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Bootsrap Grid</title>
+        <!-- Bootstrap core CSS -->
+         <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
+        <link rel="stylesheet" href="styles.css"> 
+    </head>    
+    <body>
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col">
+                    <header>
+                        <div class="container-fluid box-shadow p-3">
+                            <h5 class="my-0 mr-md-auto position-display">Company name</h5>
+                            <nav class="my-2 my-md-0 mr-md-3 nav-position">                                    
+                                <a class="p-2 text-dark" href="#">Features</a>
+                                <a class="p-2 text-dark" href="#">Enterprise</a>
+                                <a class="p-2 text-dark" href="#">Support</a>
+                                <a class="p-2 text-dark" href="#">Pricing</a>                            
+                                <button type="button" class="btn btn-outline-primary">Sign up</button>
+                            </nav>  
+                        </div>
+                    </header>    
+                </div>                                       
+            </div>
+            <div class="container pt-md-5">
+                <div class="row">
+                    <h1 class="text-center">Pricing</h1>
+                    <p class="text-center lead">Quickly build an effective pricing table for your potential customers with this Bootstrap example. It's built with default Bootstrap components and utilities with little customization.</p>
+                </div>
+            </div>
+            <div class="container">
+                <div class="row">
+                    <div class="col col-lg-3 col-md-3 col-sm-12">
```

Comment:
Make this col-lg-4, for all of these boxes.
Because 12/4 = 3 even column spaces, that stretch completely the length of the row.